index.html

```diff
@@ -53,86 +61,153 @@ <h2>Multi-Post Stories</h2>
-                <h4>Professional Art <br> Printing Data</h4>
-                <p class="secondbio">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+        </div>
+        <div class="desktopcard2 grido">
+            <img alt="" src="imgs/deskcard4.svg">
+            <div class="cardinfo">
+                <h4>Data Dashboard Healthcare</h4>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
                 <ul class="langcontainer">
                     <li class="opaquebuttons">html</li>
                     <li class="opaquebuttons">bootstrap</li>
                     <li class="opaquebuttons">ruby</li>
                 </ul>
-                <input type="button" value="See Project" class="thiccbutton">
             </div>
-        </section>
-        <section class="myworks">
-            <div class="secondcard">
-                <h4>Professional Art <br> Printing Data</h4>
-                <p class="secondbio">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+        </div>
+        <div class="desktopcard2 grido">
+            <img alt="" src="imgs/deskcard2.svg">
+            <div class="cardinfo">
+                <h4>Data Dashboard Healthcare</h4>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
                 <ul class="langcontainer">
                     <li class="opaquebuttons">html</li>
                     <li class="opaquebuttons">bootstrap</li>
                     <li class="opaquebuttons">ruby</li>
                 </ul>
-                <input type="button" value="See Project" class="thiccbutton">
             </div>
-        </section>
-        <section class="myworks">
-            <div class="secondcard">
-                <h4>Professional Art <br> Printing Data</h4>
-                <p class="secondbio">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+        </div>
+        <div class="desktopcard2 grido">
+            <img alt="" src="imgs/deskcard3.svg">
+            <div class="cardinfo">
+                <h4>Data Dashboard Healthcare</h4>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
                 <ul class="langcontainer">
                     <li class="opaquebuttons">html</li>
                     <li class="opaquebuttons">bootstrap</li>
                     <li class="opaquebuttons">ruby</li>
                 </ul>
-                <input type="button" value="See Project" class="thiccbutton">
             </div>
-
-        </section>
-    </div>
+        </div>
+    </section>
     <section class="about">
```

Comment:
Though your design is not bad, this section on your desktop version(992px - 1100px) doesn't look like the one in the design. In order for your design to be perfect please fix the spacing around your button, you should try to match the template as closely as possible which means following the guidelines for spacing around elements.

Please try adding the corresponding CSS property to stick to the design as much as possible. For example, why don't you try adding some padding to the bottom of this section?

Check the image below to have a better understanding.

![Capture2](https://user-images.githubusercontent.com/75709904/144657875-0ab6f01c-b8d2-40cf-8409-13f757cd57c7.PNG)